### PR TITLE
Update CORTX prep script

### PR DESCRIPTION
**Pull request by 83bhp**
Update CORTX prep script with:
1. Install missing packages
2. Gracefully handle errors
3. Improvise logging

Signed-off-by: Pritam Bhavsar <pritam.bhavsar@seagate.com>

### CODE REVIEW

File: srv/components/provisioner/scripts/cortx_prep.sh [hunk 1]
```diff
@@ -156,28 +174,30 @@ install_dependency_pkgs()
...
-        wget "${nodejs_tar}"
+        echo -e "\tInstalling nodejs" | tee -a "${LOG_FILE}"
+        echo -e "\tDEBUG: Downloading nodeja tarball" >> "${LOG_FILE}"
+        wget "${nodejs_tar}" >> "${LOG_FILE}" 2>&1
         mkdir /opt/nodejs
-        echo "DEBUG: Extracting the tarball" >> "${LOG_FILE}"
+        echo -e "\tDEBUG: Extracting the tarball" >> "${LOG_FILE}"
         tar -C /opt/nodejs/ -xf node-v12.13.0-linux-x64.tar.xz >> "${LOG_FILE}"
-        echo "DEBUG: The extrracted tarball is kept at /opt/nodejs, removing the tarball ${nodejs_tar}" >> "${LOG_FILE}"
+        echo -e "\tDEBUG: The extrracted tarball is kept at /opt/nodejs, removing the tarball ${nodejs_tar}" >> "${LOG_FILE}"
```
Comment (nitin-seagate) [inline srv/components/provisioner/scripts/cortx_prep.sh:198]: Typo - `extracted`

File: srv/components/provisioner/scripts/cortx_prep.sh [hunk 2]
```diff
@@ -318,4 +340,10 @@ main()
 }
 
 main $@
-echo "Cortx prep script run successfully" | tee -a "${LOG_FILE}"
+echo "\
+************************* SUCCESS!!! **************************************
+
+Successfully set up the factory environment for Cortx !!
+
+The detailed logs can be seen at: "$LOG_FILE"
```
Comment (nitin-seagate) [inline srv/components/provisioner/scripts/cortx_prep.sh:348]: `The logs are available at: "$LOG_FILE"`

**Review by nitin-seagate — APPROVED** [review]
Please test thoroughly before merging else looks good to me...

**Review by mazinamdar — APPROVED** [review]
looks good to me

**Pull request merged**